### DISCUSSION TITLE
Example not making any sense

### DISCUSSION THREAD

**Discussion by wireva** [Milkdown/milkdown#1057] [Q&A] [answered]
```ts
import { Undo, history } from '@milkdown/plugin-history';
import { callCommand } from '@milkdown/plugin-utils';

const editor = await Editor.make().use(/* ... */).use(history).create();

editor.action(callCommand(Undo))
```
there is no export called Undo on that plugin, only undoCommand and I can't figure out how to call it.

```ts
import { Editor, commandsCtx } from '@milkdown/core';
import { commonmark, ToggleItalic } from '@milkdown/preset-commonmark';

async function setup() {
  const editor = await Editor.make().use(commonmark).create();

  const toggleItalic = () =>
    editor.action((ctx) => {
      // get command manager
      const commandManager = ctx.get(commandsCtx);

      // call command
      commandManager.call(ToggleItalic);
    });

  // get markdown string:
  $button.onClick = toggleItalic;
}
```
![image](https://github.com/Milkdown/milkdown/assets/46248939/11140950-192a-4918-a7f1-6b9bdbd93d2f)
Here, ToggleItalic also does not exist. Has there been an update to how commands work that is not yet reflected in the examples or am I doing something terribly wrong?

**Comment by wireva** [comment]
https://github.com/search?q=org%3AMilkdown+ToggleItalic&type=code

When searching over the entire Milkdown organization it only finds the function in  the example

**Comment by Saul-Mirone** [comment, marked as answer]
https://github.com/Milkdown/website/commit/483918bf7488a2f370cca14b1a00b6df486422bb

**Reply by wireva** [reply]
Thank you. The docs for the undo plugin also use the wrong syntax:
https://milkdown.dev/docs/api/plugin-history#commands
